docs/src/main/sphinx/sql/show-stats.rst

```diff
@@ -23,14 +23,38 @@ table lists the returned columns and what statistics they represent. Any
 additional statistics collected on the data source, other than those listed
 here, are not included.
 
-==========================  ============================================================= =================================
-Column                      Description                                                   Notes
-==========================  ============================================================= =================================
-``column_name``             The name of the column                                        ``NULL`` in the table summary row
-``data_size``               The total size in bytes of all of the values in the column    ``NULL`` in the table summary row. Available for columns of textual types (``CHAR``, ``VARCHAR``, etc)
-``distinct_values_count``   The estimated number of distinct values in the column m       ``NULL`` in the table summary row
-``nulls_fractions``         The portion of the values in the column that are ``NULL``     ``NULL`` in the table summary row.
-``row_count``               The estimated number of rows in the table                     ``NULL`` in column statistic rows
-``low_value``               The lowest value found in this column                         ``NULL`` in the table summary row. Available for columns of numeric types (``BIGINT``, ``DECIMAL``, etc)
-``high_value``              The highest value found in this column                        ``NULL`` in the table summary row. Available for columns of numeric types (``BIGINT``, ``DECIMAL``, etc)
-==========================  ============================================================= =================================
+.. list-table:: Statistics
+  :widths: 20, 40, 40
+  :header-rows: 1
+
+  * - Column
+    - Description
+    - Notes
+  * - ``column_name``
+    - The name of the column
+    - ``NULL`` in the table summary row
+  * - ``data_size``
+    - The total size in bytes of all of the values in the column
+    - ``NULL`` in the table summary row. Available for columns of textual types
+      (``CHAR``, ``VARCHAR``, etc)
+  * - ``distinct_values_count``
+    - The estimated number of distinct values in the column
+    - ``NULL`` in the table summary row
+  * - ``nulls_fractions``
+    - The portion of the values in the column that are ``NULL``
+    - ``NULL`` in the table summary row.
+  * - ``row_count``
+    - The estimated number of rows in the table
+    - ``NULL`` in column statistic rows
+  * - ``low_value``
+    - The lowest value found in this column
+    - ``NULL`` in the table summary row. Available for columns of with
+      :ref:`date and time <date-time-data-types>`, :ref:`integer
```

Comment:
Can you confirm that date and time columns are supported @martint .. and if not all.. how do we go about documenting which ones are supported.

Also note I clarified to integer and fix-precision

Reply:
only `DATE`, no other date-time types support stats today AFAIK

Reply:
floating point is also supported for min and max.

Reply:
Thanks @hashhar  

I will update .. but question .. arent all date and time just stored as numbers and therefore "should" work ..

Reply:
not exactly, dates are stored as a number which are directly comparable since it's days since epoch. timestamps get "packed" into a number but those numbers aren't directly comparable to one another.